contingency/contingency-api/src/main/java/com/powsybl/contingency/SidedContingencyElementUtils.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.contingency;
+
+import com.powsybl.iidm.network.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.function.Function;
+
+/**
+ * @author Laurent Issertial {@literal <laurent.issertial at rte-france.com>}
+ */
+public final class SidedContingencyElementUtils {
```

Comment:
As we can (since java 8 maybe) add some static methods in the interface itself, this design `Foo` interface plus `FooUtils` for utilities is less and less used and we often prefer adding directly static utilities in the interface itself.